spec/rails_helper.rb

```diff
@@ -26,12 +22,9 @@
 Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
 
 ActiveRecord::Migration.maintain_test_schema!
-WebMock.disable_net_connect!(allow: Chewy.settings[:host], allow_localhost: RUN_SYSTEM_SPECS)
+WebMock.disable_net_connect!(allow: Chewy.settings[:host], allow_localhost: true)
```

Comment:
Can we somehow make it dependent on the tag? I think it makes sense to forbid any network connection on the non-fullstack tests. But I'm not super attached to it, I suppose this is fine too if the alternative is too much trouble.

Reply:
I was initially trying to change this constant to be set based on the rspec tags (the logic currently in streaming manager in this PR) instead of the env var, but the place it was initially set is before rspec loads so thats not available yet. I'll take a stab at moving it to be before this line but after rspec load, and see what that looks like.